### PR TITLE
Update FontAwesome

### PR DESCRIPTION
**Pull request by cmitz**
### Summary

- Replace `<i/>` with {{fa-icon}} for consistency
- Update and fix some minor style issues (e.g. some filterbars)

~As I asked in Slack, please advice:~
- ~Should we replace `ember-font-awesome` with `@fortawesome/ember-fontawesome`?~

As discussed in Slack:
- [x] Replaced `ember-font-awesome` with `@fortawesome/ember-fontawersome`

### Screenshots
Some examples of "minor fixes".

Before:

image

After:

image

p.s. this is the perfect example of "[bikeshedding](http://phk.freebsd.dk/sagas/bikeshed/)"

### CODE REVIEW

File: app/templates/components/footer-nav.hbs [hunk 1]
```diff
@@ -5,58 +5,117 @@
...
+
               <h5 class="footer-header" itemprop="name"> {{t 'C.S.V. Alpha'}} </h5>
               <meta itemprop="url" content="https://csvalpha.nl">
               <meta itemprop="logo" content="/images/alphalogo.svg">
+
               <table class="table table-sm">
                 <tbody>
                   <tr>
-                    <td class="footer-table-cell"><i class="fa fa-envelope" aria-hidden="true"></i></td>
+                    <td class="footer-table-cell">{{fa-icon 'envelope'}}</td>
```
Comment (Matthijsy) [inline app/templates/components/footer-nav.hbs]: Don't we need all the aria-hidden?

Comment (cmitz) [reply app/templates/components/footer-nav.hbs]: Those are included when using `{{fa-icon}}`. We needed those on every FA icon anyway

Comment (Matthijsy) [reply app/templates/components/footer-nav.hbs]: Ah great! LGTM!

File: Dockerfile.buildkite [hunk 2]
```diff
@@ -3,13 +3,18 @@ MAINTAINER C.S.V. Alpha <ict@csvalpha.nl>
...
+ARG FA5_TOKEN
 
 # Install chrome
 RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
 RUN echo "deb http://dl.google.com/linux/chrome/deb/ stable main" > /etc/apt/sources.list.d/google.list
 RUN apt-get update
 RUN apt-get install -y google-chrome-stable
 
+# Install NPM Token
+COPY .buildkite/ /app/.buildkite/
```
Comment (Matthijsy) [inline Dockerfile.buildkite]: Maybe don't copy the whole directory but just the build.sh file?

**Pull request merged**